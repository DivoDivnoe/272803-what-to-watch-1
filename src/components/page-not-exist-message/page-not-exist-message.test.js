import React from 'react';
import renderer from 'react-test-renderer';
import PageNotExistMessage from './page-not-exist-message.jsx';

describe(`PageNotExistMessage component`, () => {
  it(`renders correctly`, () => {
    const tree = renderer.create(
        <PageNotExistMessage />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
