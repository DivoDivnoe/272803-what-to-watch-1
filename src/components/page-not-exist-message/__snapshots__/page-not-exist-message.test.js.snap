// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageNotExistMessage component renders correctly 1`] = `
<h1>
  The requested page does not exist
</h1>
`;
